Makefile

```diff
@@ -235,9 +235,6 @@ endif
 ifndef CRC_BINARY
 	CRC_BINARY = --crc-binary=$(GOPATH)/bin
 endif
-ifndef VERSION_TO_TEST
-	VERSION_TO_TEST = --crc-version=$(CRC_VERSION)+$(COMMIT_SHA)
-endif
 e2e:
 	@go test --timeout=180m $(MODULEPATH)/test/e2e -tags "$(BUILDTAGS)" --ldflags="$(VERSION_VARIABLES)" -v $(PULL_SECRET_FILE) $(BUNDLE_LOCATION) $(CRC_BINARY) $(GODOG_OPTS) $(CLEANUP_HOME) $(VERSION_TO_TEST) $(INSTALLER_PATH) $(USER_PASSWORD)
```

Comment:
should we remove ` $(VERSION_TO_TEST)` as part of e2e target?

Reply:
Since there is a `ifndef VERSION_TO_TEST` check, the code assumes `VERSION_TO_TEST` can be overridden externally (eg `VERSION_TO_TEST="--crc-version=0.1.2" make e2e`).
If we keep it, it will expand to the empty string if unset, which is fine, and the previous behaviour will be unchanged if it's set.
Imo it's better to keep it.